packages/web-components/src/divider/divider.options.ts

```diff
@@ -0,0 +1,55 @@
+import { ValuesOf } from '@microsoft/fast-foundation';
+
+/**
+ * Align content within divider
+ * @public
+ */
+export const AlignContent = {
```

Comment:
Let's prefix this with `Divider` - as an aside, curious of @miroslavstastny's thoughts, does `DividerAlignContent` seem right here as a typing? The attribute is great but I wonder if `DividerAlignment` makes more sense. There's no strict typing for this in FUIR9 so it seems like as long as the attribute/property are aligned that we can name the type intuitively?

Reply:
Technically, this property is the alignment of content within the divider, not the divider itself.

Reply:
Good point - if we can't figure out a better name let's just be sure we prefix as it ends up globally exported.